src/main/java/org/llorllale/cactoos/matchers/MatcherEnvelope.java

```diff
@@ -40,7 +40,7 @@
  * Matcher Envelope.
  * @param <T> The type of the Matcher.
  * @since 1.0.0
- * @todo #75:30min Refactor other matchers to extend MatcherEnvelope.
+ * @todo #94:30min Refactor other matchers to extend MatcherEnvelope.
  *  If you do not know how to do it please refer to InputHasContent
  *  class as the example.
  */
```

Comment:
Since abstract class MatcherEnvelope cannot be instantiated directly, please mark constructor `MatcherEnvelope final Func<T, Boolean>...` package-private or protected.

Reply:
I made it protected.